SConstruct

```diff
@@ -871,6 +877,21 @@ for library in ( "GafferUI", ) :
 	if int( env["QT_VERSION"] ) > 4 :
 		addQtLibrary( library, "Widgets" )
 
+
+vTuneRoot = env.subst("$VTUNE_ROOT")
+
+if os.path.exists( vTuneRoot ):
+	libraries["Gaffer"] = {
```

Comment:
Overwriting `libraries["Gaffer"]` like this seems pretty likely to cause problems in the future - I doubt anyone editing the original definition is going to remember to come down here and edit this too. Can we just append to the existing definition instead?